executor/runtime/types/types.go

```diff
@@ -129,9 +129,10 @@ type SidecarContainerConfig struct {
 // ExtraContainer stores data about the other containers running alongside the
 // main container in the C&W implementation of pods
 type ExtraContainer struct {
-	Name        string           // Name of the container from the pod spec
-	ID          string           // Docker container id
-	V1Container corev1.Container // The k8s definition of the container from the pod object
+	Name        string                 // Name of the container from the pod spec
+	ID          string                 // Docker container id
```

Comment:
If containerID is a part of status, I'd prefer we drop it from here.

Reply:
good idea